packages/components/src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -26,155 +44,79 @@ const useStyles = createUseStyles({
     transition: '0.6s',
   },
   // Style classes
-  card: ({ width, height, cardPosition }) => ({
-    width: width || 600,
-    height: height || 400,
-    perspective: 1000,
-    outline: 'none',
+  card: ({ width, height }) => ({
+    width,
+    height,
+    position: 'absolute',
     overflow: 'hidden',
-    transform: `translate(${cardPosition.x}px, ${cardPosition.y}px)`,
   }),
-  flipped: { transform: 'rotateY(180deg)' },
-  cardFlipper: {
+  flipped: ({ faceUp }) => ({ transform: `rotateY(${faceUp ? 0 : 180}deg)` }),
+  cardFlipper: ({ width, height }) => ({
+    width,
+    height,
     transition: '0.6s',
     transformStyle: 'preserve-3d',
     position: 'relative',
-  },
-  cardFront: {
+  }),
+  cardFront: ({ width, height }) => ({
+    width,
+    height,
     extend: 'cardFace',
-    zIndex: 2,
     transform: 'rotateY(0deg)',
-  },
-  cardBack: {
+  }),
+  cardBack: ({ width, height }) => ({
+    width,
+    height,
     extend: 'cardFace',
     transform: 'rotateY(180deg)',
-  },
-  cardGrabbing: {
-    cursor: 'grabbing',
-  },
-  cardGrab: {
-    cursor: 'grab',
-  },
+  }),
 });
 
 /**
  * Base card
  * @param props Card props
  */
-export const BaseCard = forwardRef<CardHandle, BaseCardProps>(
-  (
-    {
-      frontFace,
-      backFace,
-      faceUp,
-      width,
-      height,
-      disableNativeEvents,
-      position = { x: 0, y: 0 },
-      onPositionChanged = () => {},
-    },
-    ref
-  ) => {
-    const [isDragging, setIsDraging] = useState<boolean>(false);
-    const [cardPosition, setCardPosition] = useState<Position>(position);
-    const [previousCardPosition, setPreviousCardPosition] = useState<Position>(position);
-    const [startPosition, setStartPosition] = useState<Position>(position);
-
-    const classes = useStyles({
-      width,
-      height,
-      cardPosition,
-    });
+export const BaseCard: FunctionComponent<BaseCardProps> = ({
+  className,
+  frontFace,
+  backFace,
+  faceUp,
+  width,
+  height,
+  boundaries,
+  disableNativeEvents,
+  position = { x: 0, y: 0 },
+  onPositionChanged = () => {},
```

Comment:
Since this is an optional callback (also on ``Draggable``), wouldn't be better to remove the default value?

Reply:
As discussed with you, I prefer this option because this way we don't have to check that the callback exists before calling it